test/client/fixtures/runner/automation/regression-test.js

```diff
@@ -732,7 +732,11 @@ $(document).ready(function () {
             '</style>'
         ].join('\n');
 
-        $(style)
+        // NOTE: we need use sandboxed jQuery to process 'style' element content. Since
+        // Hammerhead 8.0.0 proxing performed on prototypes (instead of elements before)
```

Comment:
```
        // NOTE: we need to use a sandboxed jQuery to process the 'style' element content. Since
        // Hammerhead 8.0.0, proxying is performed on prototypes (instead of elements)
```